### ISSUE TITLE
Add Carthage support [enhancement]

### ISSUE THREAD

**Issue by SimonKapl** [zjfjack/JZCalendarWeekView#70]
Currently we are using your library via CocoaPods. While trying to optimise our app start, we figured out, that using CocoaPods lead to a massiv dip in initial app start performance. The amount of pod dependencies is not really an issue, but having at least one made our performance bad.

So we moved all our dependencies from CocoaPods to Carthage and were able to improve the app start performance significantly.
Now your library is the only one left without Carthage support. Our current workaround is to clone your library and fix all the build errors manually. This should work for some time, but it will also make keeping the JZCalWeekView up to date in our project a pain in the a**. 😉 

Therefore it would be really great if you could add Carthage support. I'm sure this could also be very interesting for other developers who strive for great app performance.

**Comment by zjfjack** [comment]
Okay, I will add Carthage support this weekend.

**Comment by SimonKapl** [comment]
Thx a lot. Really appreciate that.

**Comment by zjfjack** [comment]
Hi Simon,

I tried just now. Actually, it is supported by Carthage. Could you have a try and let me know what's wrong with it.

I am quite new to Carthage, so it might have some problems. Thanks

**Comment by PhilJay** [comment]
I can confirm that we now successfully resolved & built the latest release (0.6.3) with Carthage. There must have been an issue on our end that caused our previous builds to fail or a recent change to the library resulted in it now working properly.

**TL;DR**
Carthage is now supported, this issue can be closed.

```ruby
github "zjfjack/JZCalendarWeekView" ~> 0.6.3
```
